Automattic/WP-Job-Manager
# Fix Jobs dashboard table spacing and layout

Fixes #1655

#### Changes proposed in this Pull Request:

* Add padding, min width and proper alignments to cells
* Make dashboard wider, page content width on most themes
* Add `.section-inner` wrapper for Twenty Twenty theme, which similarly sets it as a wide content block

#### Testing instructions:

* Add `[job_dashboard]` shortcode to a page, or open the pre-made Job Dashboard page
* See that table is visually pleasing and makes good use of space
* Check that nothing is horribly broken with other common themes


#### Proposed changelog entry for your changes:

* Improve Job Dashboard page theme compatibility
